examples/src/app/presence/Presence.tsx

```diff
@@ -9,13 +9,16 @@ export function Presence() {
   const myColor = useRef(COLORS[Math.floor(Math.random() * COLORS.length)])
   const [presence, setPresence] = usePresence<{ x: number; y: number; color: string }>()
 
-  const updatePresence = useCallback((e: React.MouseEvent<HTMLDivElement>) => {
-    setPresence({
-      x: e.clientX - e.currentTarget.offsetLeft,
-      y: e.clientY - e.currentTarget.offsetTop,
-      color: myColor.current,
-    })
-  }, [])
+  const updatePresence = useCallback(
+    (e: React.MouseEvent<HTMLDivElement>) => {
+      setPresence({
+        x: e.clientX - e.currentTarget.offsetLeft,
+        y: e.clientY - e.currentTarget.offsetTop,
+        color: myColor.current,
+      })
+    },
+    [setPresence],
```

Comment:
Minor: can remove `setPresence` from deps since state setters are guaranteed by React to have consistent references across renders. https://react.dev/learn/synchronizing-with-effects#why-was-the-ref-omitted-from-the-dependency-array

Reply:
Oh wait - duh - this is a setter from _our_ hook. Maybe we should make sure this stays referentially stable? 🤔 I'm leaving this in for now, though.

Reply:
Yeah, if it's not there we get a linter warning. I wonder if there is an annotation or something we can use to tell the linter that it's referentially stable?

Reply:
Yeah, the linter might not be able to work that out. But also our `setPresence` isn't technically referentially stable since it changes whenever the awareness context changes.